internal/events/eventstream.go

```diff
@@ -453,15 +450,17 @@ func (a *eventStream) eventPoller() {
 				i1 := checkpoint[sub.info.ID]
 				i2 := sub.blockHWM()
 
-				changed = changed || blockUpdatedFilterStale || i1 == nil || i1.Cmp(&i2) != 0
+				subChanged := i1 == nil || i1.Cmp(&i2) != 0
+				if subChanged {
+					log.Debugf("%s: New checkpoint HWM: %s", a.spec.ID, i2.String())
```

Comment:
Replaced other less helpful logging, to just show the one subscription that updated its checkpoint